src/client/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -302,12 +302,8 @@ public T WithExtraQueryParameters(string extraQueryParameters)
         /// </summary>
         /// <remarks>
         /// Changes in the public API of experimental features will not result in an increment of the major version of this library.
-        /// The same level of support is given to experimental features as to all other features.
-        /// Affected APIs are clearly documented as being experimental. 
-        /// For these reasons we advise against using these features in production.
+        /// For these reason we advise against using these features in production.
         /// </remarks>
-        /// <param name="enableExperimentalFeatues"></param>
-        /// <returns></returns>
         public T WithExperimentalFeatues(bool enableExperimentalFeatues = true)
```

Comment:
@bgavrilMS there is a typo here. 
WithExperimentalFeatues => WithExperimentalFeatures

Reply:
Thank you for catching this Jean-Marc, I fixed it.